.github/workflows/ci.yml

```diff
@@ -34,6 +34,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Set the tag GC interval to 1 on linux
+        if: runner.os == 'macOS'
```

Comment:
the comment refers to linux, but the condition to macOS, slightly confusing

Reply:
:facepalm: perfectly fine content for a second PR